api/v1alpha1/virtualmachine_conversion_test.go

```diff
@@ -224,8 +224,8 @@ func TestVirtualMachineConversion(t *testing.T) {
 							Name: "my-cdrom-image",
 							Kind: "VirtualMachineImage",
 						},
-						AllowGuestControl: true,
-						Connected:         true,
+						AllowGuestControl: &[]bool{true}[0],
```

Comment:
Interesting, don't think I've seen that syntax before. You could also use `ptrOf(true)` here. That func is in both v1alpha[12] packages

Reply:
I see both usages in the current code. Let me change it to calling that function since it's clearer and more readable.

Reply:
Updated in the latest commit.